# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/mkdosfs/big-endian-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mkdosfs-0.4.1/mkdosfs.c.orig	1998-09-01 23:46:30.000000000 +0200
+++ mkdosfs-0.4.1/mkdosfs.c	2003-07-13 07:11:20.000000000 +0200
@@ -78,9 +78,9 @@
 
 #define CF_LE_W(v) ((((v) & 0x00ff) << 8) | (((v) & 0xff00) >> 8))
 #define CT_LE_W(v) ((((v) & 0x00ff) << 8) | (((v) & 0xff00) >> 8))
-#define CF_LE_L(v) ((((v) & 0x0000ff00) <<  8) | (((v) & 0x00ff0000) >> 8) |
+#define CF_LE_L(v) ((((v) & 0x0000ff00) <<  8) | (((v) & 0x00ff0000) >> 8) | \
                     (((v) & 0x000000ff) << 24) | (((v) & 0xff000000) >> 24))
-#define CT_LE_L(v) ((((v) & 0x0000ff00) <<  8) | (((v) & 0x00ff0000) >> 8) |
+#define CT_LE_L(v) ((((v) & 0x0000ff00) <<  8) | (((v) & 0x00ff0000) >> 8) | \
                     (((v) & 0x000000ff) << 24) | (((v) & 0xff000000) >> 24))
 
 #else
